### DISCUSSION TITLE
bro can u please add a small feature of making the navigation outside ?

### DISCUSSION THREAD

**Discussion by viswasakthi3** [ismail9k/vue3-carousel#355] [Ideas]
having spent 100+ hours using ur awesome project i want this feature despreately can u please add it

**Comment by ismail9k** [comment]
Hello @viswasakthi3 please check this [example](https://github.com/ismail9k/vue3-carousel/blob/master/docs/examples/ExampleCustomNavigation.vue) that explains how to achieve your request
